DeepSpeech.ipynb

```diff
@@ -1387,7 +1389,8 @@
     "            avg_tower_gradients = average_gradients(tower_results[1])\n",
     "\n",
     "            # Add logging of averaged gradients\n",
-    "            log_grads_and_vars(avg_tower_gradients)\n",
+    "            if log_variables:\n",
+    "                log_grads_and_vars(avg_tower_gradients)\n",
```

Comment:
It does change the graph, as we're not adding the summary ops anymore, but I don't think it would break checkpointing or something like that. I'll test it.